src/Runner/Runner.php

```diff
@@ -176,6 +177,10 @@ private function fixFile(\SplFileInfo $file, LintingResultInterface $lintingResu
                     $tokens->clearEmptyTokens();
                     $tokens->clearChanged();
                     $appliedFixers[] = $fixer->getName();
+
+                    if (!isset($extraInfoFixers['helpUri'][$fixer->getName()])) {
+                        $extraInfoFixers['helpUri'][$fixer->getName()] = $fixer->getHelpUri();
```

Comment:
This is the only place `getHelpUrl` is called, so we can have the logic from `AbstractFixer::getHelpUri` in here and do not break BC, right?

Reply:
@kubawerlos are you familiar with [this comment](#discussion_r1597365923)? Because info about URIs would be used with another, dedicated reporter too.

Reply:
But I did not verify it against that concept, so maybe you're right it would also work for it 🙂. Just wanted to make sure you have the context.

Reply:
I saw it, and as in https://github.com/llaville/sarif-php-sdk/blob/1.2.0/src/Converter/PhpStanConverter.php the URLs are in there, not coming from PHPStan, so my understanding is that both `PhpCsFixerConverter` and `Runner` can have it (maybe shared by some new constant in PHP CS Fixer repo, but definitely not wich BC break change).

Reply:
I'll give more explains later, especially in context of SARIF reports.
@kubawerlos The `PhpCsFixerConverter` is not yet available on my repo. Will come later when this PR will be finished !